curriculum/challenges/english/25-front-end-development/lecture-working-with-css-transforms-overflow-and-filters/672bcc9c4a6dd6d7dd3e6357.md

```diff
@@ -10,6 +10,26 @@ dashedName: what-is-the-css-box-model
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+What is the CSS box model, and how does it work?
+
+The CSS box model is a fundamental concept for web development. It defines how HTML elements are structured and positioned. If you understand this model you will be able to controll the size, spacing and appearance of the elements on your website.
+
+In the CSS box model, every element is sorrounded by a box. This box consists of four elements: the content area, the padding, the border, and the margin. 
+
+The content area is the innermost part of the box. It's the space that contains the actual content of an element like text or images. You can see the content area in this representation of an HTML element in the CSS box model.
```

Comment:
not sure how to handle that last sentence because it says "you can see the content area" but there is no illustration since we are not adding images. so that might need to be updated